warehouse/ingest-csv/src/test/java/datawave/ingest/csv/TableConfigurationUtilTest.java

```diff
@@ -176,6 +184,25 @@ public void testOutputTableNotInCache() throws AccumuloSecurityException, Accumu
 
     }
 
+    @Test(expected = IOException.class)
+    public void testOutputTableNotInCache() throws AccumuloSecurityException, AccumuloException, TableNotFoundException, IOException {
+        conf.set(TableConfigCache.ACCUMULO_CONFIG_FILE_CACHE_ENABLE_PROPERTY, "true");
+        File tempCacheFile = File.createTempFile("tempCache2", null);
+        conf.set(TableConfigCache.ACCUMULO_CONFIG_CACHE_PATH_PROPERTY, tempCacheFile.getAbsolutePath());
+
+        TableConfigurationUtil tcu = new TableConfigurationUtil(conf);
+        TableConfigurationUtil.registerTableNamesFromConfigFiles(conf);
+        tcu.configureTables(conf);
+
+        tcu.updateCacheFile();
+
+        tcu.addOutputTables("nonexistent_table", conf);
+
+        tcu.serializeTableConfgurationIntoConf(conf);
+        tempCacheFile.delete();
```

Comment:
We'll actually never reach this cleanup call here. You could put it in a finally block after the `tcu.serializeTableCofnigurationIntoConf()` call.